wwwroot/config.json

```diff
@@ -1,50 +1,166 @@
+                "title": "Compare different dates at one location"
+              },
+              {
+                "markdownDescription": "- Escape the Split Screen Mode by pressing the 'x' of Split Screen Mode title in the workbench\n- Remove the duplicate copy of your dataset in the workbench using the 3 dots drop-down menu",
+                "title": "Close Split Screen Mode"
+              }
+            ],
+            "title": "View and compare different points in time"
+          },
+          {
+            "steps": [
+              {
+                "markdownDescription": "- This includes the dataset Delta Blended Service\n- Search or scroll to find the location where you want to perform change detection",
+                "title": "Add a Change Detection / Difference-enabled dataset to the map"
+              },
+              {
+                "markdownDescription": "-  Behind the 3 dots in the workbench, select the menu item 'Difference'\n-   Difference mode will activate, splitting the screen and adding new options on the left",
+                "title": "Turn on Difference mode"
+              },
+              {
+                "markdownDescription": "-   Use the date pickers at the bottom to choose your dates for screen A and B\n -   The Preview style defaults to True colour to help you find cloud-free images, change if required\n -   If you have previously searched for a location, this will be used to refine the imagery to only show available images for that location\n -   If you didn't search for a location, select a point on the map in your area of interest to refine the imagery for that area only\n -   Select your Difference Output, e.g. NDVI\n -   Select Generate Change Detection",
+                "title": "Select your Difference variables"
+              },
+              {
+                "markdownDescription": "-   The legend for the Difference calculation will appear on the left to help your analysis\n -   You can change dates using the date pickers to toggle between dates and a new Change Detection calculation will be run\n -   Close or exit Difference mode when you are finished your analysis",
+                "title": "View the change detection results on-screen"
+              }
+            ],
+            "title": "Detecting change between two points in time"
+          }
+        ]
+      }
+    ],
+    "mobileDefaultViewerMode": "2d",
+    "regionMappingDefinitionsUrl": "https://raw.githubusercontent.com/TerriaJS/saas-catalogs-public/main/nationalmap/regionMapping-2021-06-02.json",
+    "showWelcomeMessage": true,
+    "supportEmail": "info@terria.io",
```

Comment:
This email is shown in some error messages. Should we change it to `data@digital.gov.au` from above? /cc @AnaBelgun

Reply:
Not at this stage, because NationalMap is being transferred from DTA to GA.